src/main/java/uk/gov/hmcts/sscs/personalisation/Personalisation.java

```diff
@@ -71,7 +71,10 @@ public Personalisation(NotificationConfig config, MessageAuthenticationServiceIm
             personalisation.put(VENUE_MAP_LINK_LITERAL, latestHearing.getVenueGoogleMapUrl());
         }
 
-        setEvidenceProcessingAddress(personalisation, ccdResponse.getCaseReference());
+        if (config.isJobSchedulerEnabled()) {
```

Comment:
This is always false. If this is the intention then the name `isJobSchedulerEnabled` is confusing since the value is the real jobScheduler status is not connected. On the other hand, if this is just a flag to fix temporarily that npe then I'd suggest renaming it for something more readable to make reference to its purpose